opcua/gopcua/sub.go

```diff
@@ -45,26 +45,32 @@ func NewClient(ctx context.Context, svc opcua.Service, log logger.Logger) opcua.
 
 // Subscribe subscribes to the OPC-UA Server.
 func (b client) Subscribe(cfg opcua.Config) error {
-	endpoints, err := opcuaGopcua.GetEndpoints(cfg.ServerURI)
-	if err != nil {
-		return errors.Wrap(errFailedFetchEndpoint, err)
+	opts := []opcuaGopcua.Option{
```

Comment:
I let you decide, but maybe shorten the naming. Something like `gopc` or (`gopcua`) and `gua`?

Reply:
I'll keep this in mind for the next PR